entroq.go

```diff
@@ -108,12 +112,19 @@ type Task struct {
 
 	Created  time.Time `json:"created"`
 	Modified time.Time `json:"modified"`
+
+	// Worker retry logic uses these fields when moving tasks and when retrying them.
+	// It is left up to the consumer to determine how many attempts is too many
+	// and to produce a suitable retry or move error.
+	Attempt int32  `json:"attempt"`
```

Comment:
I'm unfamiliar with this syntax, so maybe you can teach me when we talk through things?

Reply:
Certainly! The syntax is called a "struct tag", and it basically annotates a field for anything that might want to do something with it via reflection. When converting structs to JSON, for example, you might not want your field names capitalized. This is how you can send a message to the built-in JSON parser/producer about how JSON names map to struct fields.